front-end/src/components/CommentFormElement.tsx

```diff
@@ -48,11 +48,11 @@ export default function CommentFormElement(props: Props) {
         setCommentAuthor(author);
         return author;
       }
-    }).then((author: any) => {
-      if (author) {
-        const comment: PostComment = {
-          author,
-          comment: commentContent,
+    }).then(() => {
+      if (commentAuthor) {
+        const comment = {
+          "author": commentAuthor,
+          "comment": commentContent,
```

Comment:
I changed PostComment to include an id, so that I can have the id of the comment when liking the comment, which is why I removed the type here.